app/src/main/res/values-land/dimens.xml

```diff
@@ -384,4 +384,10 @@
   <dimen name="profile_rename_activity_profile_rename_input_margin_end">168dp</dimen>
   <dimen name="profile_rename_activity_profile_rename_save_button_margin_end">168dp</dimen>
 
+  <!-- Profile Chooser Profile View -->
+  <dimen name="profile_chooser_profile_divider_view_layout_margin_top">28dp</dimen>
```

Comment:
@Rishavgupta12345 all these dimens in these 3 files have not been used anywhere, just take a look at the PR and changes you have made , are they actually what is required?

Reply:
use text compare to analyse the difference in portrait and landscape of the file and then make the required changes to merge and delete unused files, I did not get how have you merged the files as there was the difference in gravity and other parameters also